packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -75,26 +84,19 @@ const components = [
         {longMessage}
       </BpkBannerAlert>,
     ],
-    readme: bannerAlertReadme,
-  },
-  {
-    id: 'toasts',
-    title: 'Toasts',
-    blurb: 'Coming soon',
-    examples: [],
   },
 ];
 
-const NotificationsPage = () => <DocsPageBuilder
-  title="Notifications"
+const BannerAlertsPage = () => <DocsPageBuilder
+  title="Banner alerts"
   blurb={[
     <Paragraph>
-      Backpack supports a number of different types of notification styles, which can be used in different scenarios
-      to provide messaging to the user.
+      Banner alerts provide feedback to the user when an action has been performed.
     </Paragraph>,
   ]}
   components={components}
   sassdocId="notifications"
```

Comment:
Stills say's `notifications` - is this ok? I realise that we'd have to change token level groups / categories. Perhaps that ship has sailed.